### DISCUSSION TITLE
M1 Mac Support

### DISCUSSION THREAD

**Discussion by southsideboats** [sbender9/WilhelmSKSupport#21] [Ideas]
With the new Mac M1 chip supporting iOS apps, might be good for some people (like me) to be able to use the app on their laptop.

**Comment by sbender9** [comment]
Hmm. It should work. Are you getting an error message when you run it?

**Reply by sbender9** [reply]
Oh. You're not talking about the Mac Catalyst version?

There is a version available on the Mac App Store. 

I don't think there's anything I can do to make sideloading iOS apps work. My understanding is that Apple does not allow that.

**Reply by southsideboats** [reply]
I am unclear on this myself. On these M1 chip macs, in the app store there is a section marked "iPhone and iPad apps". According to Apple, it's "up to the developer to make the app available" whatever that means. Frankly I'm a little surprised you don't have one of these M1 machines yet. Pretty zippy little laptops. 

"Choose Whether to Include Your iOS App on the Mac App Store
After you sign the updated developer agreement, the App Store automatically makes compatible iOS apps available to users of a Mac with Apple silicon. However, if you’re already planning to ship a macOS version of your app, or if your app doesn’t make sense on the Mac, you can change your app’s availability in App Store Connect.

Open the Pricing and Availability page in App Store Connect.

Disable the Make this app available on Mac option.

Once you remove your app’s availability for Mac, the Mac App Store stops offering your app for sale. Users who already downloaded the app on Mac can still use it, but can’t download it again."

**Reply by sbender9** [reply]
I need to look into that more, I think there may be an issue because I have the Mac Catalyst version available. 

If you search for WilhelmSK in the normal App Store on your Mac, you should see that it is available?

**Reply by sbender9** [reply]
OK, I just turned this on, so it should become available. No idea if it will work ok or how long it will take to show up in the store...

**Reply by southsideboats** [reply]
Sorry I went awol on this. It's working on my M1 Mac. I have not had a chance to really mess with it much... I am talking about the iOS version here that was previously unavailable.... I will play with it and get back to you. 

I have the M1 Air, 512gb model with the 8 graphics cores and 8 gb ram. Man I swear the battery lasts like 3 days on this thing. No fan either is really nice.
